docs/guides/deployment.rst

```diff
@@ -1,27 +1,64 @@
 Deployment
 ==========
 
-Docker
-~~~~~~
+One line deploy
+~~~~~~~~~~~~~~~
 
-We continuously deploy our `Hoodie Tracker App`_ using Docker. You can read
-about our continuous
-deployment set at `hoodie-app-tracker/deployment.md`_.
+After you’ve built your Hoodie app you probably want to put it online. You can choose to deploy your app as read-only or deploy the backend couchdb database as well. This `video`_ and the text below descibes how to deploy your app using one line of code. Alternatively, you can deploy your app using Docker, please refer to the Docker section. 
 
-.. _Hoodie Tracker App: https://github.com/hoodiehq/hoodie-app-tracker
-.. _hoodie-app-tracker/deployment.md: https://github.com/hoodiehq/hoodie-app-tracker/blob/maste/deployment.md
+.. _video: https://youtu.be/29Uclxq_1Vw
+Deploying to Now
+~~~~~~~~~~~~~~~~
+.. _command line tool: https://github.com/zeit/now-cli
+`Now`_ allows you to deploy a Node application with its `command line tool`_. It’s 100% free for Open Source projects. You can deploy an app from your computer or right from a GitHub repository. For example, to deploy our Hoodie Tracker demo app all you have to do is to run this command:
```

Comment:
Could you link `Hoodie Tracker demo` to https://github.com/hoodiehq/hoodie-app-tracker ?